app/models/manageiq/providers/ibm_cloud/power_virtual_servers/regions.rb

```diff
@@ -0,0 +1,13 @@
+class ManageIQ::Providers::IbmCloud::PowerVirtualServers::Regions < ManageIQ::Providers::Regions
+  class << self
+      private
+
+      def ems_type
+          :ibm_cloud_power_virtual_servers
+      end
+
+      def regions_yml
+          ManageIQ::Providers::IbmCloud::Engine.root.join("config/power_virtual_servers_regions.yml")
+      end
```

Comment:
Almost done! 😄 All of these nested indents should only be 2 spaces each.
```suggestion
    private

    def ems_type
      :ibm_cloud_power_virtual_servers
    end

    def regions_yml
      ManageIQ::Providers::IbmCloud::Engine.root.join("config/power_virtual_servers_regions.yml")
    end
```